tensorflow-jupyter/tensorflow-deployment.yaml

```diff
@@ -60,14 +60,14 @@ spec:
       # Read more about affinity at: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
       affinity:
         nodeAffinity:
-          # This will REQUIRE the Pod to be run on a system with a Pascal GPU
+          # This will REQUIRE the Pod to be run on a system with an NVIDIA A40 GPU
```

Comment:
I added a commit to update this comment to align with the change in GPU selection